lib/js/components/ProgressBar/ProgressBar.stories.ts

```diff
@@ -0,0 +1,174 @@
+import ProgressBar from './ProgressBar.vue';
+import {
+	PROGRESS_BAR_COLORS,
+	PROGRESS_BAR_SIZES,
+	PROGRESS_BAR_LAYOUTS,
+	PROGRESS_BAR_RADII,
+	PROGRESS_BAR_LAYERS,
+	PROGRESS_BAR_COLOR_SCHEMES,
+} from './ProgressBar.consts';
+
+import { Args, ArgTypes, Meta, StoryFn } from '@storybook/vue';
+
+export default {
+	title: 'Components/ProgressBar',
+	component: ProgressBar,
+} as Meta<typeof ProgressBar>;
+
+const StoryTemplate: StoryFn<typeof ProgressBar> = (argTypes) => ({
+	components: { ProgressBar },
+	props: Object.keys(argTypes),
+	template: `<ProgressBar v-bind=$props></ProgressBar>`,
+});
+
+export const Interactive = StoryTemplate.bind({});
+
+const argTypes = {
+	numberOfLayers: {
+		control: { type: 'select', options: Object.values(PROGRESS_BAR_LAYERS) },
+		defaultValue: PROGRESS_BAR_LAYERS.ONE,
+	},
+	colorScheme: {
+		control: { type: 'select', options: Object.values(PROGRESS_BAR_COLOR_SCHEMES) },
+		defaultValue: PROGRESS_BAR_COLOR_SCHEMES.DEFAULT,
+	},
+	color: {
+		control: { type: 'select', options: Object.values(PROGRESS_BAR_COLORS) },
+		defaultValue: PROGRESS_BAR_COLORS.PRIMARY,
```

Comment:
INFO is default color
```suggestion
		defaultValue: PROGRESS_BAR_COLORS.INFO,
```

Reply:
Fixed.